pkg/appStore/deployment/service/InstalledAppService.go

```diff
@@ -780,6 +780,49 @@ func (impl *InstalledAppServiceImpl) FindAppDetailsForAppstoreApplication(instal
 	}
 	return appDetail, nil
 }
+func (impl *InstalledAppServiceImpl) FindNotesForArgoApplication(installedAppId, envId int) (string, string, error) {
+	installedAppVerison, err := impl.installedAppRepository.GetInstalledAppVersionByInstalledAppIdAndEnvId(installedAppId, envId)
+	var notes string
+	appName := installedAppVerison.InstalledApp.App.AppName
+	if err != nil {
```

Comment:
handle the error just after line 784. if there is an error while getting installedAppVersion then installedAppVersion will come as nil and code will break since you are fetching AppName from it, return empty strings while returning.